corehq/apps/data_dictionary/util.py

```diff
@@ -382,7 +382,7 @@ def is_case_type_or_prop_name_valid(case_prop_name):
     return match_obj is not None
 
 
-@quickcache(vary_on=['domain'], timeout=60 * 60)
+@quickcache(vary_on=['domain'], timeout=60 * 10)
```

Comment:
nit: is there a way to asses the impact this would have on DB with cache expiring earlier?
Its only used when loading DD right? Which I believe should be okay since that page is only available for advance plan and higher.

Reply:
The only real way I can think of for assessing this would be to do a performance test both with and without the delete feature enabled. It's worth noting however, that QA has done a performance test on this and concluded that it didn't have a noticable impact on performance.

Given the above, I think it should be fine with the reduced cache expiry.

Reply:
Thanks @zandre-eng 
Where can I see details of the performance test done? I was requesting something similar lately so would be helpful to see how QA does this.

Reply:
You can have a look at this QA ticket for more context around the [performance testing](https://dimagi.atlassian.net/browse/QA-5997) that was done.